core/pouch/index.js

```diff
@@ -112,6 +141,16 @@ class Pouch {
     })
   }
 
+  async runMigrations() {
+    log.debug('Running migrations...')
+    for (const migration of migrations) {
+      const result = await migrate(migration, this)
+      const { errors, msg } = migrationLog(migration, result)
+      log.debug({ errors }, msg)
```

Comment:
Only a debug log if the migration has some errors?

Reply:
I'd like to know when migrations are run and their result (noop, failed, complete).
In case there aren't any errors, it will print only one line per migration. This seems small enough to me.

Reply:
I'm OK with that. My point is that if there is an error, shouldn't we do something like alerting the user or sending the error to sentry?

Reply:
Errors will be logged but we should probably send them to Sentry indeed.